@xen-orchestra/immutable-backups/doc.md

```diff
@@ -0,0 +1,147 @@
+# Imutability
+
+the goal is to make a remote that XO can write, but not modify during the immutability duration set on the remote. That way, it's not possible for XO to delete or crypt any backup during this period. It protects your backup agains ransomware, at least as long as the attacker does not have a root access to the remote server.
```

Comment:
Did you mean "encrypt" instead of "crypt" ?

Reply:
I concur, eg from Kaspersky website:
> The aim of crypto ransomware is to encrypt your important data